framework/console/controllers/FixtureController.php

```diff
@@ -482,6 +482,8 @@ private function getFixturesConfig($fixtures)
                 $config[] = $fullClassName;
             } elseif (class_exists($fullClassName . 'Fixture')) {
                 $config[] = $fullClassName . 'Fixture';
+            } else {
+                throw new Exception( 'Fixtures '.$fullClassName. ' or ' . $fullClassName . 'Fixture' . " were not found".($isNamespaced ?" in namespace: ' . $this->namespace . '" :'.'));
```

Comment:
Wouldn't if..else-ing the message make it more readable than inline if. Other than that I think it fine